fe/fe-core/src/main/java/org/apache/doris/analysis/CreateTableStmt.java

```diff
@@ -715,12 +714,16 @@ public String toSql() {
             }
         }
         sb.append("\n)");
-        sb.append(" ENGINE = ").append(engineName);
+        sb.append(" ENGINE = ").append(engineName.toUpperCase());
```

Comment:
use toLowerCase

Reply:
It has been modified to toLowerCase, and the code format has been adjusted using the project's IntelliJ-code-format.